kafka-ui-react-app/src/components/Schemas/List/List.tsx

```diff
@@ -1,45 +1,115 @@
 import ClusterContext from 'components/contexts/ClusterContext';
+import ConfirmationModal from 'components/common/ConfirmationModal/ConfirmationModal';
+import { useForm } from 'react-hook-form';
 
 import ListItem from './ListItem';
 
 export interface ListProps {
   schemas: SchemaSubject[];
   isFetching: boolean;
+  isGlobalSchemaCompatibilityLevelFetched: boolean;
+  globalSchemaCompatibilityLevel?: CompatibilityLevelCompatibilityEnum;
   fetchSchemasByClusterName: (clusterName: ClusterName) => void;
+  fetchGlobalSchemaCompatibilityLevel: (
+    clusterName: ClusterName
+  ) => Promise<void>;
+  updateGlobalSchemaCompatibilityLevel: (
+    clusterName: ClusterName,
+    compatibilityLevel: CompatibilityLevelCompatibilityEnum
+  ) => Promise<void>;
 }
 
 const List: React.FC<ListProps> = ({
   schemas,
   isFetching,
+  globalSchemaCompatibilityLevel,
+  isGlobalSchemaCompatibilityLevelFetched,
   fetchSchemasByClusterName,
+  fetchGlobalSchemaCompatibilityLevel,
+  updateGlobalSchemaCompatibilityLevel,
 }) => {
   const { isReadOnly } = React.useContext(ClusterContext);
   const { clusterName } = useParams<{ clusterName: string }>();
+  const {
+    register,
+    handleSubmit,
+    formState: { isSubmitting },
+  } = useForm();
+  const [
+    isUpdateCompatibilityConfirmationVisible,
+    setUpdateCompatibilityConfirmationVisible,
+  ] = React.useState(false);
 
   React.useEffect(() => {
     fetchSchemasByClusterName(clusterName);
+    fetchGlobalSchemaCompatibilityLevel(clusterName);
   }, [fetchSchemasByClusterName, clusterName]);
 
+  const onCompatibilityLevelUpdate = async ({
+    compatibilityLevel,
+  }: {
+    compatibilityLevel: CompatibilityLevelCompatibilityEnum;
+  }) => {
+    await updateGlobalSchemaCompatibilityLevel(clusterName, compatibilityLevel);
+    setUpdateCompatibilityConfirmationVisible(false);
+  };
+
   return (
     <div className="section">
       <Breadcrumb>Schema Registry</Breadcrumb>
       <div className="box">
         <div className="level">
-          {!isReadOnly && (
+          {!isReadOnly && isGlobalSchemaCompatibilityLevelFetched && (
             <div className="level-item level-right">
+              <h5 className="is-5 mr-2">Global Compatibility Level: </h5>
+              <div className="select mr-2">
+                <select
+                  name="compatibilityLevel"
+                  defaultValue={globalSchemaCompatibilityLevel}
+                  ref={register()}
+                  onChange={() =>
+                    setUpdateCompatibilityConfirmationVisible(true)
+                  }
+                >
+                  {Object.keys(CompatibilityLevelCompatibilityEnum).map(
+                    (level: string) => (
+                      <option key={level} value={level}>
+                        {level}
+                      </option>
+                    )
+                  )}
+                </select>
+              </div>
```

Comment:
Can you please create a new component for this? That will make List component much cleaner.